tests/test.js

```diff
@@ -0,0 +1,48 @@
+
+function videoCount(divWidth) {
+    let videoCount;
+    if (divWidth < 500) {
+        videoCount = 1;
+    } else if (divWidth < 900) {
+        videoCount = 2;
+    } else if (divWidth < 1200) {
+        videoCount = 3;
+    } else if (divWidth < 1500) {
+        videoCount = 4;
+    } else {
+        videoCount = 5;
+    }
+
+    return videoCount;
+}
+
+describe("Case", function () {
+    it("Succesfully returned video count", function () {
+        expect(videoCount(1200)).toBe(4);
```

Comment:
you should write the test cases for the script.js code which you are using for your app.

Reply:
Resolved